pkg/pmem-csi-driver/pmem-csi-driver.go

```diff
@@ -295,7 +295,7 @@ func (pmemd *pmemDriver) Run() error {
 		if pmemd.cfg.Mode == Node {
 			klog.V(3).Info("Unregistering node...")
 			if err := pmemd.unregisterNodeController(); err != nil {
-				klog.V(4).Infof("Failed to node unregister: %v", err)
+				klog.V(3).Infof("Failed to unregister node: %v", err)
```

Comment:
@avalluri why were these info messages and not `klog.Errorf` messages?

Reply:
replaced with klog.Errorf

Reply:
> @avalluri why were these info messages and not `klog.Errorf` messages?

No specific reason I could recall. It could be left-over of a copy-paste.